quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/FeaturesDistTest.java

```diff
@@ -33,7 +33,7 @@ public class FeaturesDistTest {
     private static final String PREVIEW_FEATURES_EXPECTED_LOG = "Preview features enabled: " + Arrays.stream(Profile.Feature.values())
             .filter(feature -> feature.getType() == Profile.Feature.Type.PREVIEW)
             .filter(feature -> {
-                Set<Profile.Feature> versions = Profile.getFeatureVersions(feature.getKey());
+                Set<Profile.Feature> versions = Profile.getFeatureVersions(feature.getUnversionedKey());
```

Comment:
Why is this change required?

Reply:
I believe this is a bug. The method is calling `getOrderedFeatures` which returns map of `f.getUnversionedKey()` -> to all versions.

Therefore using versioned key here does not make sense and never finds anything. There was no versioned feature in preview mode at the moment so me changing `ROLLING_UDPATES_V2` to preview triggered the bug.

Reply:
Thanks for explaining. Please create an issue to track the bug.